python/py-rpds_py/Portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-rpds_py
```

Comment:
this port should be called `py-rpds-py` as is done on PyPI and as it is listed on upstream's GitHub page. To make sure it can find the download file, which is confusingly, names `rpds_py`. You should add `distname rpsd_py-${version}`.

Reply:
Thanks for this! I was indeed wondering how to fix the file/port name discrepancy,